api

```diff
@@ -1889,6 +1898,54 @@ get_device_info() { #returns information about current install and hardware
   
 }
 
+get_model() {
```

Comment:
Is this function specifically for Jetson devices? In any case, it's probably best to explain its usage in a comment.

Reply:
its a partial copy of the get_device_info function but with extra stuff that slowed it down removed.

the below part is just for detecting if we are running on a jetson, yes, which I wrote originally for the megascript and retropie.

Reply:
added with https://github.com/Botspot/pi-apps/pull/1890/commits/081e8219c3b273432acc677795e6ed362019c8ef

Reply:
While better, this function does not have a summary stating what it does or how it's used.
I assume that it is not outputting to stdout because you set the `$model` _and_ `$jetson_model` variables?

Reply:
right, the idea was this would be a function that could just be used to get some identifying model information for the hardware the user is running. right now its just the generic model and specifically the jetson_model (if applicable)

but it could be extended as necessary kinda like what is done over at retropie for the systems they support: 
https://github.com/RetroPie/RetroPie-Setup/blob/8e2fda6a2273222f50a39b490cec85cfbc3cba2f/scriptmodules/system.sh#L349-L437